File: app/src/main/java/com/bakkenbaeck/sol/service/TimeReceiver.java
package com.bakkenbaeck.sol.service;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class TimeReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        final Intent service = new Intent(context, SunsetService.class);
        context.startService(service);
    }
}

File: app/src/main/java/com/bakkenbaeck/sol/service/TimeReceiver.kt
package com.bakkenbaeck.sol.service

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class TimeReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val service = Intent(context, SunsetService::class.java)
        context.startService(service)
    }
}
